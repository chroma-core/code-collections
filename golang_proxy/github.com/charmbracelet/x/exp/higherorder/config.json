{
    "native_identifier": "github.com/charmbracelet/x/exp/higherorder",
    "repo": "charmbracelet/x",
    "registry": "golang_proxy",
    "tag_formats": [
        "/v{major}.{minor}.{patch}",
        "ansi/v{major}.{minor}.{patch}",
        "ansi/v{major}.{patch}.{minor}"
    ],
    "sentinel_timestamp": "2020-01-01T00:00:00Z",
    "include": [
        "exp/higherorder/**/CMakeLists.txt",
        "exp/higherorder/**/Dockerfile",
        "exp/higherorder/**/Dockerfile.*",
        "exp/higherorder/**/Makefile",
        "exp/higherorder/**/GNUmakefile",
        "exp/higherorder/**/BSDmakefile",
        "exp/higherorder/**/Makefile.*",
        "exp/higherorder/**/go.mod",
        "exp/higherorder/**/go.sum",
        "exp/higherorder/**/Cargo.toml",
        "exp/higherorder/**/Cargo.lock",
        "exp/higherorder/**/Gemfile",
        "exp/higherorder/**/Rakefile",
        "exp/higherorder/**/Vagrantfile",
        "exp/higherorder/**/Procfile",
        "exp/higherorder/**/.babelrc",
        "exp/higherorder/**/.eslintrc",
        "exp/higherorder/**/.prettierrc",
        "exp/higherorder/**/.editorconfig",
        "exp/higherorder/**/.gitconfig",
        "exp/higherorder/**/.npmrc",
        "exp/higherorder/**/.env",
        "exp/higherorder/**/.env.*",
        "exp/higherorder/**/.condarc",
        "exp/higherorder/**/*.go",
        "exp/higherorder/**/*.rs",
        "exp/higherorder/**/*.py",
        "exp/higherorder/**/*.js",
        "exp/higherorder/**/*.mjs",
        "exp/higherorder/**/*.cjs",
        "exp/higherorder/**/*.jsx",
        "exp/higherorder/**/*.ts",
        "exp/higherorder/**/*.mts",
        "exp/higherorder/**/*.cts",
        "exp/higherorder/**/*.tsx",
        "exp/higherorder/**/*.xml",
        "exp/higherorder/**/*.xsd",
        "exp/higherorder/**/*.xsl",
        "exp/higherorder/**/*.xslt",
        "exp/higherorder/**/*.rss",
        "exp/higherorder/**/*.atom",
        "exp/higherorder/**/*.xaml",
        "exp/higherorder/**/*.plist",
        "exp/higherorder/**/*.svg",
        "exp/higherorder/**/*.dtd",
        "exp/higherorder/**/*.md",
        "exp/higherorder/**/*.markdown",
        "exp/higherorder/**/*.mdown",
        "exp/higherorder/**/*.mkdn",
        "exp/higherorder/**/*.mkd",
        "exp/higherorder/**/*.rmd",
        "exp/higherorder/**/*.mdx",
        "exp/higherorder/**/*.yaml",
        "exp/higherorder/**/*.yml",
        "exp/higherorder/**/*.json",
        "exp/higherorder/**/*.jsonc",
        "exp/higherorder/**/*.json5",
        "exp/higherorder/**/*.geojson",
        "exp/higherorder/**/*.har",
        "exp/higherorder/**/*.toml",
        "exp/higherorder/**/*.ini",
        "exp/higherorder/**/*.cfg",
        "exp/higherorder/**/*.conf",
        "exp/higherorder/**/*.editorconfig",
        "exp/higherorder/**/*.properties",
        "exp/higherorder/**/*.sh",
        "exp/higherorder/**/*.bash",
        "exp/higherorder/**/*.ksh",
        "exp/higherorder/**/*.zsh",
        "exp/higherorder/**/*.sql",
        "exp/higherorder/**/*.ipynb",
        "exp/higherorder/**/*.c",
        "exp/higherorder/**/*.h",
        "exp/higherorder/**/*.cc",
        "exp/higherorder/**/*.cpp",
        "exp/higherorder/**/*.cxx",
        "exp/higherorder/**/*.hh",
        "exp/higherorder/**/*.hpp",
        "exp/higherorder/**/*.hxx",
        "exp/higherorder/**/*.ipp",
        "exp/higherorder/**/*.m",
        "exp/higherorder/**/*.mm",
        "exp/higherorder/**/*.cs",
        "exp/higherorder/**/*.java",
        "exp/higherorder/**/*.kt",
        "exp/higherorder/**/*.kts",
        "exp/higherorder/**/*.scala",
        "exp/higherorder/**/*.sc",
        "exp/higherorder/**/*.sbt",
        "exp/higherorder/**/*.swift",
        "exp/higherorder/**/*.php",
        "exp/higherorder/**/*.phtml",
        "exp/higherorder/**/*.php3",
        "exp/higherorder/**/*.php4",
        "exp/higherorder/**/*.php5",
        "exp/higherorder/**/*.phps",
        "exp/higherorder/**/*.rb",
        "exp/higherorder/**/*.gemspec",
        "exp/higherorder/**/*.r",
        "exp/higherorder/**/*.lua",
        "exp/higherorder/**/*.css",
        "exp/higherorder/**/*.scss",
        "exp/higherorder/**/*.sass",
        "exp/higherorder/**/*.vue",
        "exp/higherorder/**/*.svelte",
        "exp/higherorder/**/*.hcl",
        "exp/higherorder/**/*.tf",
        "exp/higherorder/**/*.tfvars",
        "exp/higherorder/**/*.nomad",
        "exp/higherorder/**/*.proto",
        "exp/higherorder/**/*.cmake",
        "exp/higherorder/**/*.mk",
        "exp/higherorder/**/*.make",
        "exp/higherorder/**/*.s",
        "exp/higherorder/**/*.asm",
        "exp/higherorder/**/*.dart",
        "exp/higherorder/**/*.hs",
        "exp/higherorder/**/*.lhs",
        "exp/higherorder/**/*.txt",
        "exp/higherorder/**/*.text",
        "exp/higherorder/**/*.log",
        "exp/higherorder/**/*.nix"
    ],
    "collection_name_prefix": "github.com_charmbracelet_x_exp_higherorder",
    "version_sample_relative_size": 0.5,
    "version_sample_max_size": 5
}
