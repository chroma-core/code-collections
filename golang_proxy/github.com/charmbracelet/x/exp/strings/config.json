{
  "native_identifier": "github.com/charmbracelet/x/exp/strings",
  "repo": "charmbracelet/x",
  "registry": "golang_proxy",
  "tag_formats": [
    "/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.0",
    "ansi/v{major}.{patch}.0"
  ],
  "sentinel_timestamp": "2020-01-01T00:00:00Z",
  "include": [
    "exp/strings/**/CMakeLists.txt",
    "exp/strings/**/Dockerfile",
    "exp/strings/**/Dockerfile.*",
    "exp/strings/**/Makefile",
    "exp/strings/**/GNUmakefile",
    "exp/strings/**/BSDmakefile",
    "exp/strings/**/Makefile.*",
    "exp/strings/**/go.mod",
    "exp/strings/**/go.sum",
    "exp/strings/**/Cargo.toml",
    "exp/strings/**/Cargo.lock",
    "exp/strings/**/Gemfile",
    "exp/strings/**/Rakefile",
    "exp/strings/**/Vagrantfile",
    "exp/strings/**/Procfile",
    "exp/strings/**/.babelrc",
    "exp/strings/**/.eslintrc",
    "exp/strings/**/.prettierrc",
    "exp/strings/**/.editorconfig",
    "exp/strings/**/.gitconfig",
    "exp/strings/**/.npmrc",
    "exp/strings/**/.env",
    "exp/strings/**/.env.*",
    "exp/strings/**/.condarc",
    "exp/strings/**/*.go",
    "exp/strings/**/*.rs",
    "exp/strings/**/*.py",
    "exp/strings/**/*.js",
    "exp/strings/**/*.mjs",
    "exp/strings/**/*.cjs",
    "exp/strings/**/*.jsx",
    "exp/strings/**/*.ts",
    "exp/strings/**/*.mts",
    "exp/strings/**/*.cts",
    "exp/strings/**/*.tsx",
    "exp/strings/**/*.xml",
    "exp/strings/**/*.xsd",
    "exp/strings/**/*.xsl",
    "exp/strings/**/*.xslt",
    "exp/strings/**/*.rss",
    "exp/strings/**/*.atom",
    "exp/strings/**/*.xaml",
    "exp/strings/**/*.plist",
    "exp/strings/**/*.svg",
    "exp/strings/**/*.dtd",
    "exp/strings/**/*.md",
    "exp/strings/**/*.markdown",
    "exp/strings/**/*.mdown",
    "exp/strings/**/*.mkdn",
    "exp/strings/**/*.mkd",
    "exp/strings/**/*.rmd",
    "exp/strings/**/*.mdx",
    "exp/strings/**/*.yaml",
    "exp/strings/**/*.yml",
    "exp/strings/**/*.json",
    "exp/strings/**/*.jsonc",
    "exp/strings/**/*.json5",
    "exp/strings/**/*.geojson",
    "exp/strings/**/*.har",
    "exp/strings/**/*.toml",
    "exp/strings/**/*.ini",
    "exp/strings/**/*.cfg",
    "exp/strings/**/*.conf",
    "exp/strings/**/*.editorconfig",
    "exp/strings/**/*.properties",
    "exp/strings/**/*.sh",
    "exp/strings/**/*.bash",
    "exp/strings/**/*.ksh",
    "exp/strings/**/*.zsh",
    "exp/strings/**/*.sql",
    "exp/strings/**/*.ipynb",
    "exp/strings/**/*.c",
    "exp/strings/**/*.h",
    "exp/strings/**/*.cc",
    "exp/strings/**/*.cpp",
    "exp/strings/**/*.cxx",
    "exp/strings/**/*.hh",
    "exp/strings/**/*.hpp",
    "exp/strings/**/*.hxx",
    "exp/strings/**/*.ipp",
    "exp/strings/**/*.m",
    "exp/strings/**/*.mm",
    "exp/strings/**/*.cs",
    "exp/strings/**/*.java",
    "exp/strings/**/*.kt",
    "exp/strings/**/*.kts",
    "exp/strings/**/*.scala",
    "exp/strings/**/*.sc",
    "exp/strings/**/*.sbt",
    "exp/strings/**/*.swift",
    "exp/strings/**/*.php",
    "exp/strings/**/*.phtml",
    "exp/strings/**/*.php3",
    "exp/strings/**/*.php4",
    "exp/strings/**/*.php5",
    "exp/strings/**/*.phps",
    "exp/strings/**/*.rb",
    "exp/strings/**/*.gemspec",
    "exp/strings/**/*.r",
    "exp/strings/**/*.lua",
    "exp/strings/**/*.css",
    "exp/strings/**/*.scss",
    "exp/strings/**/*.sass",
    "exp/strings/**/*.vue",
    "exp/strings/**/*.svelte",
    "exp/strings/**/*.hcl",
    "exp/strings/**/*.tf",
    "exp/strings/**/*.tfvars",
    "exp/strings/**/*.nomad",
    "exp/strings/**/*.proto",
    "exp/strings/**/*.cmake",
    "exp/strings/**/*.mk",
    "exp/strings/**/*.make",
    "exp/strings/**/*.s",
    "exp/strings/**/*.asm",
    "exp/strings/**/*.dart",
    "exp/strings/**/*.hs",
    "exp/strings/**/*.lhs",
    "exp/strings/**/*.txt",
    "exp/strings/**/*.text",
    "exp/strings/**/*.log",
    "exp/strings/**/*.nix"
  ],
  "collection_name_prefix": "github.com_charmbracelet_x_exp_strings",
  "version_sample_relative_size": 0.5,
  "version_sample_max_size": 5
}
