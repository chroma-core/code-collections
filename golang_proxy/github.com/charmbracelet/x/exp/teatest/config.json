{
  "native_identifier": "github.com/charmbracelet/x/exp/teatest",
  "repo": "charmbracelet/x",
  "registry": "golang_proxy",
  "tag_formats": [
    "/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.{patch}"
  ],
  "sentinel_timestamp": "2020-01-01T00:00:00Z",
  "include": [
    "exp/teatest/**/CMakeLists.txt",
    "exp/teatest/**/Dockerfile",
    "exp/teatest/**/Dockerfile.*",
    "exp/teatest/**/Makefile",
    "exp/teatest/**/GNUmakefile",
    "exp/teatest/**/BSDmakefile",
    "exp/teatest/**/Makefile.*",
    "exp/teatest/**/go.mod",
    "exp/teatest/**/go.sum",
    "exp/teatest/**/Cargo.toml",
    "exp/teatest/**/Cargo.lock",
    "exp/teatest/**/Gemfile",
    "exp/teatest/**/Rakefile",
    "exp/teatest/**/Vagrantfile",
    "exp/teatest/**/Procfile",
    "exp/teatest/**/.babelrc",
    "exp/teatest/**/.eslintrc",
    "exp/teatest/**/.prettierrc",
    "exp/teatest/**/.editorconfig",
    "exp/teatest/**/.gitconfig",
    "exp/teatest/**/.npmrc",
    "exp/teatest/**/.env",
    "exp/teatest/**/.env.*",
    "exp/teatest/**/.condarc",
    "exp/teatest/**/*.go",
    "exp/teatest/**/*.rs",
    "exp/teatest/**/*.py",
    "exp/teatest/**/*.js",
    "exp/teatest/**/*.mjs",
    "exp/teatest/**/*.cjs",
    "exp/teatest/**/*.jsx",
    "exp/teatest/**/*.ts",
    "exp/teatest/**/*.mts",
    "exp/teatest/**/*.cts",
    "exp/teatest/**/*.tsx",
    "exp/teatest/**/*.xml",
    "exp/teatest/**/*.xsd",
    "exp/teatest/**/*.xsl",
    "exp/teatest/**/*.xslt",
    "exp/teatest/**/*.rss",
    "exp/teatest/**/*.atom",
    "exp/teatest/**/*.xaml",
    "exp/teatest/**/*.plist",
    "exp/teatest/**/*.svg",
    "exp/teatest/**/*.dtd",
    "exp/teatest/**/*.md",
    "exp/teatest/**/*.markdown",
    "exp/teatest/**/*.mdown",
    "exp/teatest/**/*.mkdn",
    "exp/teatest/**/*.mkd",
    "exp/teatest/**/*.rmd",
    "exp/teatest/**/*.mdx",
    "exp/teatest/**/*.yaml",
    "exp/teatest/**/*.yml",
    "exp/teatest/**/*.json",
    "exp/teatest/**/*.jsonc",
    "exp/teatest/**/*.json5",
    "exp/teatest/**/*.geojson",
    "exp/teatest/**/*.har",
    "exp/teatest/**/*.toml",
    "exp/teatest/**/*.ini",
    "exp/teatest/**/*.cfg",
    "exp/teatest/**/*.conf",
    "exp/teatest/**/*.editorconfig",
    "exp/teatest/**/*.properties",
    "exp/teatest/**/*.sh",
    "exp/teatest/**/*.bash",
    "exp/teatest/**/*.ksh",
    "exp/teatest/**/*.zsh",
    "exp/teatest/**/*.sql",
    "exp/teatest/**/*.ipynb",
    "exp/teatest/**/*.c",
    "exp/teatest/**/*.h",
    "exp/teatest/**/*.cc",
    "exp/teatest/**/*.cpp",
    "exp/teatest/**/*.cxx",
    "exp/teatest/**/*.hh",
    "exp/teatest/**/*.hpp",
    "exp/teatest/**/*.hxx",
    "exp/teatest/**/*.ipp",
    "exp/teatest/**/*.m",
    "exp/teatest/**/*.mm",
    "exp/teatest/**/*.cs",
    "exp/teatest/**/*.java",
    "exp/teatest/**/*.kt",
    "exp/teatest/**/*.kts",
    "exp/teatest/**/*.scala",
    "exp/teatest/**/*.sc",
    "exp/teatest/**/*.sbt",
    "exp/teatest/**/*.swift",
    "exp/teatest/**/*.php",
    "exp/teatest/**/*.phtml",
    "exp/teatest/**/*.php3",
    "exp/teatest/**/*.php4",
    "exp/teatest/**/*.php5",
    "exp/teatest/**/*.phps",
    "exp/teatest/**/*.rb",
    "exp/teatest/**/*.gemspec",
    "exp/teatest/**/*.r",
    "exp/teatest/**/*.lua",
    "exp/teatest/**/*.css",
    "exp/teatest/**/*.scss",
    "exp/teatest/**/*.sass",
    "exp/teatest/**/*.vue",
    "exp/teatest/**/*.svelte",
    "exp/teatest/**/*.hcl",
    "exp/teatest/**/*.tf",
    "exp/teatest/**/*.tfvars",
    "exp/teatest/**/*.nomad",
    "exp/teatest/**/*.proto",
    "exp/teatest/**/*.cmake",
    "exp/teatest/**/*.mk",
    "exp/teatest/**/*.make",
    "exp/teatest/**/*.s",
    "exp/teatest/**/*.asm",
    "exp/teatest/**/*.dart",
    "exp/teatest/**/*.hs",
    "exp/teatest/**/*.lhs",
    "exp/teatest/**/*.txt",
    "exp/teatest/**/*.text",
    "exp/teatest/**/*.log",
    "exp/teatest/**/*.nix"
  ],
  "collection_name_prefix": "github.com_charmbracelet_x_exp_teatest",
  "version_sample_relative_size": 0.5,
  "version_sample_max_size": 5
}
