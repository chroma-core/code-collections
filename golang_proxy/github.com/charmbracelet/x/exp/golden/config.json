{
    "native_identifier": "github.com/charmbracelet/x/exp/golden",
    "repo": "charmbracelet/x",
    "registry": "golang_proxy",
    "tag_formats": [
        "/v{major}.{minor}.{patch}",
        "ansi/v{major}.{minor}.{patch}",
        "ansi/v{major}.{patch}.{minor}"
    ],
    "sentinel_timestamp": "2020-01-01T00:00:00Z",
    "include": [
        "exp/golden/**/CMakeLists.txt",
        "exp/golden/**/Dockerfile",
        "exp/golden/**/Dockerfile.*",
        "exp/golden/**/Makefile",
        "exp/golden/**/GNUmakefile",
        "exp/golden/**/BSDmakefile",
        "exp/golden/**/Makefile.*",
        "exp/golden/**/go.mod",
        "exp/golden/**/go.sum",
        "exp/golden/**/Cargo.toml",
        "exp/golden/**/Cargo.lock",
        "exp/golden/**/Gemfile",
        "exp/golden/**/Rakefile",
        "exp/golden/**/Vagrantfile",
        "exp/golden/**/Procfile",
        "exp/golden/**/.babelrc",
        "exp/golden/**/.eslintrc",
        "exp/golden/**/.prettierrc",
        "exp/golden/**/.editorconfig",
        "exp/golden/**/.gitconfig",
        "exp/golden/**/.npmrc",
        "exp/golden/**/.env",
        "exp/golden/**/.env.*",
        "exp/golden/**/.condarc",
        "exp/golden/**/*.go",
        "exp/golden/**/*.rs",
        "exp/golden/**/*.py",
        "exp/golden/**/*.js",
        "exp/golden/**/*.mjs",
        "exp/golden/**/*.cjs",
        "exp/golden/**/*.jsx",
        "exp/golden/**/*.ts",
        "exp/golden/**/*.mts",
        "exp/golden/**/*.cts",
        "exp/golden/**/*.tsx",
        "exp/golden/**/*.xml",
        "exp/golden/**/*.xsd",
        "exp/golden/**/*.xsl",
        "exp/golden/**/*.xslt",
        "exp/golden/**/*.rss",
        "exp/golden/**/*.atom",
        "exp/golden/**/*.xaml",
        "exp/golden/**/*.plist",
        "exp/golden/**/*.svg",
        "exp/golden/**/*.dtd",
        "exp/golden/**/*.md",
        "exp/golden/**/*.markdown",
        "exp/golden/**/*.mdown",
        "exp/golden/**/*.mkdn",
        "exp/golden/**/*.mkd",
        "exp/golden/**/*.rmd",
        "exp/golden/**/*.mdx",
        "exp/golden/**/*.yaml",
        "exp/golden/**/*.yml",
        "exp/golden/**/*.json",
        "exp/golden/**/*.jsonc",
        "exp/golden/**/*.json5",
        "exp/golden/**/*.geojson",
        "exp/golden/**/*.har",
        "exp/golden/**/*.toml",
        "exp/golden/**/*.ini",
        "exp/golden/**/*.cfg",
        "exp/golden/**/*.conf",
        "exp/golden/**/*.editorconfig",
        "exp/golden/**/*.properties",
        "exp/golden/**/*.sh",
        "exp/golden/**/*.bash",
        "exp/golden/**/*.ksh",
        "exp/golden/**/*.zsh",
        "exp/golden/**/*.sql",
        "exp/golden/**/*.ipynb",
        "exp/golden/**/*.c",
        "exp/golden/**/*.h",
        "exp/golden/**/*.cc",
        "exp/golden/**/*.cpp",
        "exp/golden/**/*.cxx",
        "exp/golden/**/*.hh",
        "exp/golden/**/*.hpp",
        "exp/golden/**/*.hxx",
        "exp/golden/**/*.ipp",
        "exp/golden/**/*.m",
        "exp/golden/**/*.mm",
        "exp/golden/**/*.cs",
        "exp/golden/**/*.java",
        "exp/golden/**/*.kt",
        "exp/golden/**/*.kts",
        "exp/golden/**/*.scala",
        "exp/golden/**/*.sc",
        "exp/golden/**/*.sbt",
        "exp/golden/**/*.swift",
        "exp/golden/**/*.php",
        "exp/golden/**/*.phtml",
        "exp/golden/**/*.php3",
        "exp/golden/**/*.php4",
        "exp/golden/**/*.php5",
        "exp/golden/**/*.phps",
        "exp/golden/**/*.rb",
        "exp/golden/**/*.gemspec",
        "exp/golden/**/*.r",
        "exp/golden/**/*.lua",
        "exp/golden/**/*.css",
        "exp/golden/**/*.scss",
        "exp/golden/**/*.sass",
        "exp/golden/**/*.vue",
        "exp/golden/**/*.svelte",
        "exp/golden/**/*.hcl",
        "exp/golden/**/*.tf",
        "exp/golden/**/*.tfvars",
        "exp/golden/**/*.nomad",
        "exp/golden/**/*.proto",
        "exp/golden/**/*.cmake",
        "exp/golden/**/*.mk",
        "exp/golden/**/*.make",
        "exp/golden/**/*.s",
        "exp/golden/**/*.asm",
        "exp/golden/**/*.dart",
        "exp/golden/**/*.hs",
        "exp/golden/**/*.lhs",
        "exp/golden/**/*.txt",
        "exp/golden/**/*.text",
        "exp/golden/**/*.log",
        "exp/golden/**/*.nix"
    ],
    "collection_name_prefix": "github.com_charmbracelet_x_exp_golden",
    "version_sample_relative_size": 0.5,
    "version_sample_max_size": 5
}
